lib/data/wrangler.py

```diff
@@ -17,8 +17,7 @@
 
 def readData(fileLoc=None):
     data = np.load(fileLoc)
-    print(data.keys())
-    return (data[item] for item in data.keys())
+    return data
```

Comment:
The reason for the change is that key ordering was non-deterministic. Could be a python2/3 thing. I think we can add it back once we moved to running in Docker. 

Incidentally, this is how I have been running, via the Makefile e.g. `make all_classifiers`